### ISSUE TITLE
Bug with for loops

### ISSUE THREAD

**Issue by targos** [tkh44/babel-plugin-sitrep#9]
Thanks for this very useful plugin!
I found that it doesn't work when there is a 'for of" loop in the function:

```js
// sitrep
function test() {
    var arr = [0, 1, 2];
    var sum = 0;
    for (const value of arr) {
        sum += value;
    }
    return sum;
}

console.log(test());
```

```console
$ babel sitrep.js --plugins sitrep
Error: sitrep.js: We don't know what to do with this node type. We were previously a Statement but we can't fit in here?
    at NodePath.insertAfter (node_modules/babel-cli/node_modules/babel-traverse/lib/path/modification.js:175:13)
    at VariableDeclaration.decls.forEach.dec (node_modules/babel-plugin-sitrep/src/index.js:95:16)
    at Array.forEach (<anonymous>)
    at VariableDeclaration (node_modules/babel-plugin-sitrep/src/index.js:75:15)
    at NodePath._call (node_modules/babel-cli/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (node_modules/babel-cli/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (node_modules/babel-cli/node_modules/babel-traverse/lib/path/context.js:105:12)
    at TraversalContext.visitQueue (node_modules/babel-cli/node_modules/babel-traverse/lib/context.js:150:16)
    at TraversalContext.visitSingle (node_modules/babel-cli/node_modules/babel-traverse/lib/context.js:108:19)
    at TraversalContext.visit (node_modules/babel-cli/node_modules/babel-traverse/lib/context.js:192:19)
```

**Comment by targos** [comment]
It also happens with regular for loops:
```js
// sitrep
function test() {
    var sum = 0;
    for (var i = 0; i < 5; i++) {
        sum += i;
    }
    return sum;
}

console.log(test());
```

**Comment by tkh44** [comment]
Thanks for the issue. I didn't even think about this case 😅

**Comment by tkh44** [comment]
Fix released in `v1.2.1`

**Comment by targos** [comment]
Thanks!
